src/components/PressableWithSecondaryInteraction/index.js

```diff
@@ -50,11 +51,12 @@ class PressableWithSecondaryInteraction extends Component {
 
     render() {
         const defaultPressableProps = _.omit(this.props, ['onSecondaryInteraction', 'children', 'onLongPress']);
+        const pressableStyle = this.props.style && StyleUtils.combineStyles(this.props.style);
 
         // On Web, Text does not support LongPress events thus manage inline mode with styling instead of using Text.
         return (
             <Pressable
-                style={this.props.inline && styles.dInline}
+                style={this.props.inline ? styles.dInline : pressableStyle}
```

Comment:
```suggestion
                style={StyleUtils.combineStyles(this.props.inline ? styles.dInline : this.props.style)}
```

Could you please check if this works?